src/Cameras/Inputs/babylon.arcrotatecamera.input.pointers.ts

```diff
@@ -70,7 +72,9 @@ module BABYLON {
                     //but emptying completly pointers collection is required to fix a bug on iPhone : 
                     //when changing orientation while pinching camera, one pointer stay pressed forever if we don't release all pointers  
                     //will be ok to put back pointers.remove(evt.pointerId); when iPhone bug corrected
-                    pointers.empty();
+                    pointers.clear();
+                    pointersOrder = new Array<number>();

```

Comment:
We cannot afford having a new array built on each frame (or input). This will stress the GC. You should use a SmartArray